### PR TITLE
docs(plutono): readme formatting

### PR DESCRIPTION
**Pull request by richardtief**
# Submit a pull request

Thank you for submitting a pull request!  
To speed up the review process, please ensure that everything below
is true:

1. This is not a duplicate of an existing Plugin.
2. No existing features have been broken without good reason.
3. The Documentation has been updated to reflect your changes.
4. Tests have been added or updated to reflect your changes.
5. All tests pass.

---

Replace any ":question:" below with information about your pull request.

## Pull Request Details

Provide details about your pull request and what it adds, fixes, or changes.

:question:

## Breaking Changes

Describe what features are broken by this pull request and why, if any.

:question:

## Issues Fixed

Enter the issue numbers resolved by this pull request below, if any.

1. :question:

## Other Relevant Information

Provide any other important details below.

:question:

### CODE REVIEW

File: plutono/README.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 title: Plutono
 ---
 
-Installs the web dashboarding system [Plutono](https://github.com/credativ/plutono)to Easily collect, correlate, and visualize Prometheus metrics with beautiful dashboards.
+Installs the web dashboarding system [Plutono](https://github.com/credativ/plutono) to Easily collect, correlate, and visualize Prometheus metrics with beautiful dashboards.
```
Comment (viennaa) [inline plutono/README.md:5]: Removing assessment.  

```suggestion
Installs the web dashboarding system [Plutono](https://github.com/credativ/plutono) to easily collect, correlate, and visualize Prometheus metrics with dashboards.
```

**Review by viennaa — CHANGES_REQUESTED** [review]
Thanks, little nit inside.

Also, shouldn't the commit message be `docs(plutono):...`?

**Review by viennaa — APPROVED** [review]
And go

**Review by ibakshay — APPROVED** [review]
LGTM!

**Pull request merged**